### DISCUSSION TITLE
Fluentautocomplete : how to auto fill data when i want to edit 1 data.

### DISCUSSION THREAD

**Discussion by ifaizah** [microsoft/fluentui-blazor#2543] [Q&A]
When I edit data, how do I make Fluent Auto Complete also fill in the data directly from the database?

**Comment by ifaizah** [comment]
![image](https://github.com/user-attachments/assets/fc2649c4-d025-4dcb-93b5-cacf859fe578)

actually this select has data, but cannot display for edit page.

**Comment by ifaizah** [comment]
<FluentAutocomplete TOption="ms_data"
                    AutoComplete="off"
                    Autofocus="true"
                    Label="Nationality"
                    Placeholder="Select Nationality"
                    OnOptionsSearch="@OnSearchAsyncNationality"
                    OptionText="@(item => item.description)"
                    OptionValue="@(item => item.code)"
                    MaximumSelectedOptions="1"
                    Style="width:100%;"
                    Id="Nationality" />
                    
                    string? SelectedNationalityValue = "";
                    
                    SelectedNationalityValue = Content.nationality;

**Comment by vnbaaij** [comment]
Sorry, it is unclear to me what you are trying to achieve.

**Reply by ifaizah** [reply]
so actually I have 1 data, let's say agent data.
then this data has a nationality field, then this 1 agent has an IDN nationality.
but, when I click the edit button, then all the data should appear in the edit form, but specifically the data for fluentautocomplete doesn't want to enter the form.
the selected value is not filled in the fluentautocomplete form.

**Reply by ifaizah** [reply]
`<FluentAutocomplete` TOption="ms_data"
                    AutoComplete="off"
                    Autofocus="true"
                    Label="Nationality"
                    Placeholder="Select Nationality"
                    OnOptionsSearch="@OnSearchAsyncNationality"
                    OptionText="@(item => item.description)"
                    OptionValue="@(item => item.code)"
                    MaximumSelectedOptions="1"
                    Style="width:100%;"
                    Id="Nationality"
                    @bind-Value = "@SelectedNationalityValue"/>
                    
                    public string? SelectedNationalityValue { get; set; }
                    
                    var result = clientMaintenanceDetail.GetDataClientByClientCode("/ClientMaintenance/GetClientByClientCode", client_code);

if (result != null)
{
    Content = result;
    SelectedNationalityValue = Content.nationality;
}
else
{
    Content = null!;
}`

**Reply by vnbaaij** [reply]
You need to specify an `Items` parameter to the FluentAutoComplete to give it the items that you can choose from. Check the examples in the demo site.
